Semantic HTML: Add `<mark>` and `<time>` tags (#396)
Codecademy/docs: content/html/concepts/semantic-html/terms/mark/mark.md

---
Title: '<mark>'

> Description: 'Represents part of a text that should be rendered as marked or highlighted, due to its being particularly relevant in its specific context. It can be used to draw the reader's attention to a specific word, phrase or section.'

Reviewer: ```suggestion
Description: 'Represents part of a text that should be rendered as marked or highlighted.'
```

---
Title: '<mark>'

> Description: "Represents part of a text that should be rendered as marked or highlighted, due to its being particularly relevant in its specific context. It can be used to draw the reader's attention to a specific word, phrase or section."

Reviewer: ```suggestion
Description: 'Represents part of a text that should be rendered as marked or highlighted, due to its being particularly relevant in its specific context. It can be used to draw the reader's attention to a specific word, phrase or section.'
```

Subjects:
  - 'Web Development'
  - 'Web Design'
Tags:

>   - 'HTML'
>   - 'Semantic'
>   - 'Syntax'
>   - 'Tags'

Reviewer: ```suggestion
  - 'Tags'
```

CatalogContent:
  - 'learn-html'
  - 'paths/front-end-engineer-career-path'
---

> The `<mark>` semantic tag represents part of a text that should be rendered as marked or highlighted, due to its being particularly relevant in its specific context. It can be used to draw the reader's attention to a specific word, phrase or section.

Reviewer: ```suggestion
The `<mark>` semantic tag represents part of a text that should be rendered as marked or highlighted, due to being particularly relevant in its specific context. It can be used to draw the reader's attention to a specific word, phrase, or section.
```

## Syntax

```html
<mark>
  <!-- Mark content goes inside the tag -->
</mark>
```

Using the `<mark>` element requires an opening and closing tag that wraps around a block of text. By default, browsers will render it as a black font on a yellow background, but this can be restyled via CSS.

> ## Example of Highlighting Key Concepts

Reviewer: ```suggestion
## Example
```
